llvm/lib/MC/MCCASObjectV1.cpp

```diff
@@ -1079,7 +1159,10 @@ void MCCASBuilder::startSection(const MCSection *Sec) {
   if (RelocLocation == Atom) {
     // Build a map for lookup.
     for (auto R : ObjectWriter.getRelocations()[Sec]) {
-      if (R.F)
+      // For the Dwarf Line Section, just append the relocations to the
+      // SectionRelocs. No Atoms are considered for this section.
+      if (R.F &&
+          Asm.getContext().getObjectFileInfo()->getDwarfLineSection() != Sec)
```

Comment:
We're calling this in a loop, we should hoist this in case it's not a cheap computation

Reply:
That is fair, though I would have imagined that the compiler might optimize for that? I will take it out of the loop regardless, good catch!